### DISCUSSION TITLE
Account Hacked and Suspended, NO RESPONSE FROM GITHUB SUPPORT AFTER 60 DAYS !

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#118728] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. I am writing to you because I am in desperate need of assistance regarding my GitHub account. Around 50 days ago, my account was maliciously accessed by an unauthorized individual. Regrettably, this hacker has not only changed the password but also the associated email address, leaving me with no access to my account whatsoever.

I have been tirelessly attempting to regain access to my account, but to no avail. The hacker's actions have completely locked me out, and I fear for the safety and integrity of my account and the projects stored within it.

My account and the projects contained within it are integral to my work, and their security has been compromised.
Your prompt attention to this matter would be greatly appreciated.

Thank you for your understanding and assistance.

My Ticket : #2593578

![51days](https://github.com/community/community/assets/56465197/6abe08ba-18a5-481f-a4b9-2e126fc18e95)

**Comment by ebndev** [comment]
Hi [asmawi117](https://github.com/asmawi117),

Thank you for being a member of the GitHub community and for your patience. We're sorry to hear about your situation. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place with the support team.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.I see that you have posted in other categories as well and will be deleting those posts.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding.
